index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="main.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="menu">
+            <h3>John Doe</h3>
+            <div><img src="./imgs/Union.svg" alt="hmb-button"></div>
+        </nav>
+        <div class="body-class">
+            <h1>Hey There. I’m Alex</h1>
+            <span>I’am a Software Developer</span>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+           <div class="social_media">
+            <img class="icon" alt="social_icon" src="imgs/gi.svg">
+            <img class="icon" alt="social_icon" src="imgs/in.svg">  
+            <img class="icon" alt="social_icon" src="imgs/Enabled.svg">
+            <img class="icon" alt="social_icon" src="imgs/Vector.svg">
+            <img class="icon" alt="social_icon" src="imgs/Disabled.svg">
+        </div>
+        </div>
```

Comment:
- [ ]  You should use the `ul` tag to group them and li per icon; each icon should have a link (`a `tag). This will make you code more clean and clear